src/runtime/Makefile

```diff
@@ -204,6 +210,7 @@ DEFDISABLEBLOCK := false
 DEFSHAREDFS_CLH_VIRTIOFS := virtio-fs
 DEFSHAREDFS_QEMU_VIRTIOFS := virtio-fs
 DEFSHAREDFS_QEMU_TDX_VIRTIOFS := virtio-9p
+DEFSHAREDFS_QEMU_SNP_VIRTIOFS := virtio-9p
```

Comment:
Shall we keep a `QEMU_TEE_VIRTIOFS`, considering that 9p will be used for all of those?
Note, it's more a question than a suggestion.

Reply:
Currently they are all the same, but I'm not sure it will remain that way. We are trying to move away from `virtio-9p` and I think the support for `virtio-fs` is going to be different on different platforms and maybe even between SEV and SEV-SNP.